documentation/modules/managing/con-custom-resources-status.adoc

```diff
@@ -83,7 +85,7 @@ spec:
   # ...
 status:
   conditions: <1>
-  - lastTransitionTime: 2019-07-23T23:46:57+0000
+  - lastTransitionTime: _YEAR_-_MONTH_-23T23:46:57+0000
```

Comment:
What is the value of this? I think you should keep the whole original timestamp or use something like `_DATE_AND_TIME_` instead of the whole timestamp.

Reply:
Thanks. I reverted and changed the year to 2021